conans/client/command.py

```diff
@@ -499,12 +499,21 @@ def build(self, *args):
                             help="Optional. Local folder containing the conaninfo.txt and "
                                  "conanbuildinfo.txt files (from a previous conan install "
                                  "execution). Defaulted to --build-folder")
+        parser.add_argument("-c", "--configure", default=False, action="store_true",
+                            help="Execute the configuration step (variable should_configure=True)")
```

Comment:
`help="Execute the configuration step (variable should_configure=True). When specified, build/install step won't run unless --build/--install specified"`

Reply:
What about this comments in the help?

Reply:
Yes, changing them, it was just the idea for the API.